###################error########################


どちらも手が込んでいて、私ごときには簡単には表現出来ない感じですね　f^_^;
お値段¥お高めですが、納得の出来です 
 list index out of range
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 43, in __init__
    self._setPAS(juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 79, in _setPAS
    tag.pas = Pas(tag.tag_id, self)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 87, in __init__
    self.__set_args(tag_predicate.features.get("項構造"), CaseInfoFormat.PASv41)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 186, in __set_args
    for items in self.__parse_case_analysis_items(analysis_result, case_info_format):
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 181, in __parse_case_analysis_items
    yield self.__parse_case_info_format(items, case_info_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 145, in __parse_case_info_format
    eid = int(items[3])
IndexError: list index out of range
###########################################
###################error########################

ボリュームがあり、間違いなく格別な味わいのモンブランなのですが、先に頂いた『ロアジス』があまりにも美味しすぎたため、感動が薄くなってしまいました(^^;;

何はともあれ、ハイレベルのケーキのみが並ぶパティスリーなので、全種類制覇してみたいと思います 
 list index out of range
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 43, in __init__
    self._setPAS(juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 79, in _setPAS
    tag.pas = Pas(tag.tag_id, self)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 87, in __init__
    self.__set_args(tag_predicate.features.get("項構造"), CaseInfoFormat.PASv41)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 186, in __set_args
    for items in self.__parse_case_analysis_items(analysis_result, case_info_format):
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 181, in __parse_case_analysis_items
    yield self.__parse_case_info_format(items, case_info_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 145, in __parse_case_info_format
    eid = int(items[3])
IndexError: list index out of range
###########################################
###################error########################

久しぶりのパリセヴェイユ...♪*ﾟ

ズラッと並んだケーキは圧巻です♥❤︎* 
 Error: ;; Invalid input <* * * 特殊 1 記号 5 * 0 * 0 NIL
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 40, in __init__
    self.parse(spec)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 106, in parse
    raise Exception("Error: %s" % string)
Exception: Error: ;; Invalid input <* * * 特殊 1 記号 5 * 0 * 0 NIL
###########################################
###################error########################


サントノーレは濃厚なキャラメルクリームも美味しいのですが、
プチシューのキャラメリゼされた部品がカリッカリで美味しい〜♬
キャラメルクリームの甘さと苦味のバランスが素晴らしい.｡*ﾟ+.*.｡　　

ご馳走さまでした



綺麗な店内　
超人気店
おしゃれスイーツ　 
 Error: ;; Invalid input <* * * 特殊 1 記号 5 * 0 * 0 NIL
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 40, in __init__
    self.parse(spec)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 106, in parse
    raise Exception("Error: %s" % string)
Exception: Error: ;; Invalid input <* * * 特殊 1 記号 5 * 0 * 0 NIL
###########################################
###################error########################
珍しくランチがケーキな日｡
自由が丘の人気店に正午前につくと,まだ空いていたので待たずに入れました｡
テイクアウトのお客さんは沢山いましたけどね｡
皆様のレビューを参考に目星をつけていたケーキを注文｡普段食べないので,その場の簡単な説明では迷います(^_^;
ケーキ名を記憶してないのですが,全体的に柑橘類やカシス 
 list index out of range
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 43, in __init__
    self._setPAS(juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 79, in _setPAS
    tag.pas = Pas(tag.tag_id, self)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 87, in __init__
    self.__set_args(tag_predicate.features.get("項構造"), CaseInfoFormat.PASv41)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 186, in __set_args
    for items in self.__parse_case_analysis_items(analysis_result, case_info_format):
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 181, in __parse_case_analysis_items
    yield self.__parse_case_info_format(items, case_info_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 145, in __parse_case_info_format
    eid = int(items[3])
IndexError: list index out of range
###########################################
###################error########################
私の中で、自由が丘といえば、スイーツ
とってもお洒落なケーキ屋さんです♪並んでいるケーキがどれも美味しそうで…迷った^_^;
甘くて、幸せな気分になれる味 
 list index out of range
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 43, in __init__
    self._setPAS(juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 79, in _setPAS
    tag.pas = Pas(tag.tag_id, self)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 84, in __init__
    for items in self.__parse_case_analysis_items(case_analysis, CaseInfoFormat.CASE):
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 181, in __parse_case_analysis_items
    yield self.__parse_case_info_format(items, case_info_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 137, in __parse_case_info_format
    tid = int(items[3])
IndexError: list index out of range
###########################################
###################error########################
質な素材　+　味の感覚が鋭いパティシエ　+　美的センス　+　立地の良さ…この方程式の答えは、何度も通いたくなるパティスリーとなります 
 Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 40, in __init__
    self.parse(spec)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 106, in parse
    raise Exception("Error: %s" % string)
Exception: Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
###########################################
